WordPress/Classes/ViewRelated/System/Floating Create Button/CreateButtonCoordinator.swift

```diff
@@ -92,6 +83,7 @@ import WordPressFlux
         super.init()
 
         listenForQuickStart()
+        fetchBloggingPrompt()
```

Comment:
This would also end up being called on the 'Pages' screen since it uses this coordinator. It could probably be mitigated by either checking the presenting view controller or the number of actions. Or alternatively, moving the fetch to the same spot the header is created. What are your thoughts?

Reply:
> This would also end up being called on the 'Pages' screen since it uses this coordinator. 

Woops. Forgot that was different.

> moving the fetch to the same spot the header is created

That was my original intent. However, since everything was dependent on the fetching, it led to a blackhole of completion blocks. The intent for fetching the prompt in `init` is to be sure there is a prompt before it is needed.

> checking the presenting view controller 

This felt a bit more fragile as things could move, the FAB could be added somewhere else, etc.

> the number of actions

This felt like the best dynamic option. Door #3 it is!